.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "docs/.tmp/ansible"]
+	path = docs/.tmp/ansible
+	url = https://github.com/ansible/ansible.git
```

Comment:
You can relate to submodules using relative paths. I think, this is nicer if they are meant to live on the same server.